doc/user/bgp.rst

```diff
@@ -2004,31 +2014,58 @@ Displaying BGP Information
 
          Total number of prefixes 1
 
-.. index:: show ip bgp regexp LINE
-.. clicmd:: show ip bgp regexp LINE
+Some other commands are available on that format, and permit to dump either IPv4
+routing table or IPv6 routing table, depending on the presence or not of the
+`ip` keyword.
+
+.. index:: show [ip] bgp regexp LINE
+.. clicmd:: show [ip] bgp regexp LINE
 
    This command displays BGP routes using AS path regular expression
    (:ref:`bgp-regular-expressions`).
 
-.. index:: show bgp <ipv4|ipv6> summary
-.. clicmd:: show bgp <ipv4|ipv6> summary
+.. index:: show [ip] bgp summary
+.. clicmd:: show [ip] bgp summary
 
    Show a bgp peer summary for the specified address family.
 
-.. index:: show bgp <ipv4|ipv6> neighbor [PEER]
-.. clicmd:: show bgp <ipv4|ipv6> neighbor [PEER]
+The old command structure :cliref:`show ip bgp` may become deprecated in the
+future and should no longer be used. In order to reach the other BGP routing
+tables other than the IPv6 routing table given by :cliref:`show bgp`, the new
+command structure is extended with :cliref:`show bgp [afi] [safi]`.
+
+.. index:: show bgp [afi] [safi]
+.. clicmd:: show bgp [afi] [safi]
+
+.. index:: show bgp <ipv4|ipv6> <unicast|multicast|vpn|labeled-unicast>
+.. clicmd:: show bgp <ipv4|ipv6> <unicast|multicast|vpn|labeled-unicast>
+
+   These commands display BGP routes for the specific routing table pointed by
+   the selected afi and the selected safi. If not afi and no safi value is
+   given, the command falls back to the default IPv6 routing table
+
+.. index:: show bgp [afi] [safi] summary
+.. clicmd:: show bgp [afi] [safi] summary
+
+   Show a bgp peer summary for the specified address family, and subsequent
+   address-family.
+
+.. index:: show bgp [afi] [safi] neighbor [PEER]
+.. clicmd:: show bgp [afi] [safi] neighbor [PEER]
 
-   This command shows information on a specific BGP `peer`.
+   This command shows information on a specific BGP `peer` of the relevant
+   afi and safi selected.
 
-.. index:: show bgp <ipv4|ipv6> dampening dampened-paths
-.. clicmd:: show bgp <ipv4|ipv6> dampening dampened-paths
+.. index:: show bgp [afi] [safi] dampening dampened-paths
+.. clicmd:: show bgp [afi] [safi] dampening dampened-paths
 
-   Display paths suppressed due to dampening.
+   Display paths suppressed due to dampening of the relevant afi and safi
+   selected.
 
-.. index:: show bgp <ipv4|ipv6> dampening flap-statistics
-.. clicmd:: show bgp <ipv4|ipv6> dampening flap-statistics
+.. index:: show bgp [afi] [safi] dampening flap-statistics
+.. clicmd:: show bgp [afi] [safi] dampening flap-statistics
 
-   Display flap statistics of routes.
+   Display flap statistics of routes of the relevant afi and safi selected.
```

Comment:
`...routes of the selected afi and safi.`

Reply:
ok